packages/runtime/container-runtime/src/batchTracker.ts

```diff
@@ -75,6 +75,6 @@ export class BatchTracker {
 export const BindBatchTracker = (
     batchEventEmitter: EventEmitter,
     logger: ITelemetryLogger,
-    batchLengthThreshold: number = 128,
+    batchLengthThreshold: number = 500,
```

Comment:
FYI  - 500 is 92th percentile right now. So that change will reduce number of errors 12.5x.
So we will still be getting 640 errors / day.
P99 is at 938, which is really close to limits (our ops are at least 400 bytes, I think).
Let's go with 1K.

Reply:
And I think it goes without saying that we need to somehow understand why we have so many ops per batch, and do we actually see critical failures. The max I see recorded here is 1669. Which likely hits the limit (it calculates to 600 bytes/op, assuming single op above it hits the limit). Which is not that far from what I'd expect (given all the overhead of JSON serialization twice).

I wonder if we should be recording these stats on batch creation/sent, not receiving. As failures would happen on send - we would not see such batches if they never left machine. What do you think?